bot/context.js

```diff
@@ -38,10 +38,10 @@ module.exports = {
 
 	loggedReply(html, extra) {
 		if (chats.adminLog) {
-			this.tg
+			this.telegram
```

Comment:
I think forwardMessage for replies should move here, and the following log should actually reply to the forwarded message:

```TS
const message = reply && await ctx.forwardMessage(...);
this.telegram.sendMessage(
	chats.adminLog,
	html,
	{ parse_mode: 'HTML', reply_to_message: message?.message_id },
).catch(() => null);
```